README.md

```diff
@@ -1,26 +1,101 @@
 
-## Ruby version
 
-## System dependencies
+## Prerequisites
 
-## Configuration
+Ensure you have the following installed on your local development machine:
 
-## Database creation
+- Ensure that Ruby >= 3.0.0 is installed.You may find the Ruby documentation helpful to have on hand. [Ruby Documentation](https://ruby-doc.org/)
+- Make sure the ruby version at the top of Gemfile matches your local ruby version (e.g. if you have ruby 3.1.2, your Gemfile should have ruby "3.1.2 at the top.Run ruby -v in the terminal to check the version you are using. In the Gemfile, verify that ruby '<version>' below source 'https://rubygems.org' is correct
 
-## Database initialization
+- Check your Rails version. If no rails, run gem install rails.
+- Check your Bundler version. If bundle -v fails, run gem install bundler to install it. 
+- Verify the Heroku command line tool has been installed in the development environment. [Heroku CLI Documentation](https://devcenter.heroku.com/articles/heroku-cli#install-with-ubuntu-debian-apt-get)
+- PostgreSQL: The default Rails development environment expects to use the SQLite3 database. For shifting to PostgreSQL for development, follow the instructions in this [guide](https://github.com/sidbav/public-health-app-v2/wiki/Shifting-to-PostgreSQL-for-development).
+
+
+## Installing
+
+Clone the repository:
+`git clone https://github.com/sidbav/public-health-app-v2.git`
+
+Install the dependencies:
+bundle install
+
+Verify you have Ruby, Rails, and Heroku CLI installed:
+ruby -v &&\
+rails -v &&\
+bundle -v &&\
+heroku -v &&\
+
+### Setup the database:
+`rails db:create`
+`rails db:migrate`
+
+
+### To seed the database, run:
+`rails db:seed`
+
+### Running the Application
+You can start the rails server using the command given below.
+`rails s`
+
+### Change the database for production
+- All apps on Heroku use the PostgreSQL database. For Ruby/Rails apps to do so, they must include the pg gem.To specify production-specific gems, add `gem 'pg'`  for Heroku deployment
+- To make sure sqlite3 gem is only loaded in development and test environments but not production.
+ make sure  `gem 'sqlite3'` is added for development
+
+- Always Run bundle install to update dependencies after you modify gemfile
+
+If you encounter an error, run `bundle config set --local without 'production'` and then `bundle install` again.Commit the updated Gemfile and Gemfile.lock.
+
+Note: Heroku ignores config/database.yml and enforces the use of PostgreSQL. Modifying the production: section in database.yml will not affect Heroku's behavior.
+
+You may find `PostgreSQL Explorer` VS code extension useful 
```

Comment:
add link to this to the extension. https://marketplace.visualstudio.com/items?itemName=ckolkman.vscode-postgres